Kickstarter-iOS/Tests/Views/DiscoveryFiltersViewControllerTests.swift

```diff
@@ -82,9 +83,8 @@ internal final class DiscoveryFiltersViewControllerTests: TestCase {
       |> SelectableRow.lens.params.staffPicks .~ true
 
     Language.allLanguages.forEach { language in
-      withEnvironment(language: language) {
-        let controller = DiscoveryFiltersViewController.configuredWith(selectedRow: staffPicksRow,
-          categories: categories)
+      withEnvironment(cache: self.cache, language: language) {
```

Comment:
this threw me off, don't understand why I had to explicitly set the cache here after modifying it in `setUp`, but this didn't work otherwise. Thoughts?